Public/Src/Utilities/KeyValueStore/KeyValueStoreAccessor.cs

```diff
@@ -308,6 +342,9 @@ private struct VersionConstants
         /// <param name="openBulkLoad">
         /// Have RocksDb open for bulk loading.
         /// </param>
+        /// <param name="invalidationHandler">
+        /// <see cref="m_invalidationHandler"/>

```

Comment:
nit: Since m_invalidationHandler is private, doesn't it make sense to just copy the description instead of referencing the private member?

Reply:
Will address this in a separate PR, as I'm going to move these into a separate configuration object.